helm/argocd-test/action.yml

```diff
@@ -0,0 +1,42 @@
+name: "Test that an helm chart renders"
+description: "Test that an helm chart renders"
+
+inputs:
+  chart-dir:
+    description: "Specify the chart directory."
+    required: true
+    default: "argocd/base/"
+  helm-option:
+    description: "Specify the options to pass to helm"
+    required: true
+    default: "-f placeholder.yaml"
+  github-token:
+    description: "Specify the github token to login to ghcr.io"
+    required: true
+
+
+runs:
+  using: "composite"
+  steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
```

Comment:
Do you need history ? If not, you can remove this as it will slow down the retrieval, especially on bug projects like the Gate